### DISCUSSION TITLE
Implementing Python code in snippets

### DISCUSSION THREAD

**Discussion by ElNadav** [artisticat1/obsidian-latex-suite#233] [Q&A] [answered]
Hi everyone, I'm new to obsidian and Latex-Suite in particular. I read Gilles Castel's article about using snippets and noticed he used Python to create the "Box" snippet. 
![Screenshot 2023-12-06 035056](https://github.com/artisticat1/obsidian-latex-suite/assets/73067218/dd737722-ed8c-4f3c-8a12-3c990caedd38)
 
I searched through the discussions and issues and didn't find anything mentioning using Python snippets in Latex-suite. I would appreciate if someone could explain to me how to use python in a snippet of my own (if that's even possible).

**Comment by superle3** [comment, marked as answer]
> I searched through the discussions and issues and didn't find anything mentioning using Python snippets in Latex-suite. I would appreciate if someone could explain to me how to use python in a snippet of my own (if that's even possible).

It wasn't until v1.9.0 with javascript. Use that instead, python will not be supported (like its very impractical to do and javascript works "good enough")
